bokehjs/src/lib/models/layouts/flex_box.ts

```diff
@@ -68,7 +68,24 @@ export abstract class FlexBoxView extends LayoutDOMView {
         }
       })()
 
-      view.parent_style.append(":host", {flex, align_self})
+      function min_size(size: number | null): string {
+        if (size != null) {
+          return px(size)
+        } else {
+          switch (sizing.width_policy) {
```

Comment:
Not fully convinced this is sufficient. The problem I was seeing was that elements with the max policy didn't stretch unless min_width: 0 was set.

Reply:
Thanks